rules/windows/driver_load/driver_load_vuln_drivers.yml

```diff
@@ -0,0 +1,803 @@
+            - '86a8e0aa29a5b52c84921188cc1f0eca9a7904dcfe09544602933d8377720219' #BS_Flash64.sys
+            - '1d0397c263d51e9fc95bcc8baf98d1a853e1c0401cd0e27c7bf5da3fba1c93a8' #BS_HWMIO64_W10.sys
+            - '60c6f4f34c7319cb3f9ca682e59d92711a05a2688badbae4891b1303cd384813' #BS_HWMIo64.sys
+            - '55fee54c0d0d873724864dc0b2a10b38b7f40300ee9cae4d9baaf8a202c4049a' #BS_I2c64.sys
+            - '61a1bdddd3c512e681818debb5bee94db701768fc25e674fcad46592a3259bd0' #GLCKIO2.sys
+            - '42f0b036687cbd7717c9efed6991c00d4e3e7b032dc965a2556c02177dfdad0f' #GVCIDrv64.sys
+            - 'bb1135b51acca8348d285dc5461d10e8f57260e7d0c8cc4a092734d53fc40cbc' #HwOs2Ec10x64.sys
+            - 'b179e1ab6dc0b1aee783adbcad4ad6bb75a8a64cb798f30c0dd2ee8aaf43e6de' #HwOs2Ec7x64.sys
+            - '525d9b51a80ca0cd4c5889a96f857e73f3a80da1ffbae59851e0f51bdfb0b6cd' #MsIo64.sys
+            - '3f2fda9a7a9c57b7138687bbce49a2e156d6095dddabb3454ea09737e02c3fa5' #NBIOLib_X64.sys
+            - '314384b40626800b1cde6fbc51ebc7d13e91398be2688c2a58354aa08d00b073' #NCHGBIOS2x64.sys
+            - 'd8b58f6a89a7618558e37afc360cd772b6731e3ba367f8d58734ecee2244a530' #NTIOLib_X64.sys
+            - '65db1b259e305a52042e07e111f4fa4af16542c8bacd33655f753ef642228890' #PhlashNT.sys
+            - '19a212e6fc324f4cb9ee5eba60f5c1fc0191799a4432265cbeaa3307c76a7fc0' #Phymemx64.sys
+            - 'a7c8f4faf3cbb088cac7753d81f8ec4c38ccb97cd9da817741f49272e8d01200' #UCOREW64.sys
+            - '677c0b1add3990fad51f492553d3533115c50a242a919437ccb145943011d2bf' #WinFlash64.sys
+            - 'fc22977ff721b3d718b71c42440ee2d8a144f3fbc7755e4331ddd5bcc65158d2' #amifldrv64.sys
+            - 'ad40e6d0f77c0e579fb87c5106bf6de3d1a9f30ee2fbf8c9c011f377fa05f173' #atillk64.sys
+            - '18e1707b319c279c7e0204074088cc39286007a1cf6cb6e269d5067d8d0628c6' #dbk64.sys
+            - 'c9cf1d627078f63a36bbde364cd0d5f2be1714124d186c06db5bcdf549a109f8' #mtcBSv64.sys
+            - 'afdd66562dea51001c3a9de300f91fc3eb965d6848dfce92ccb9b75853e02508' #nvflash.sys
+            - 'a899b659b08fbae30b182443be8ffb6a6471c1d0497b52293061754886a937a3' #nvflsh64.sys
+            - '1963d5a0e512b72353953aadbe694f73a9a576f0241a988378fa40bf574eda52' #phymem64.sys
+            - '7133a461aeb03b4d69d43f3d26cd1a9e3ee01694e97a0645a3d8aa1a44c39129' #rtkio64.sys
+            - '32e1a8513eee746d17eb5402fb9d8ff9507fb6e1238e7ff06f7a5c50ff3df993' #rtkiow10x64.sys
+            - '082c39fe2e3217004206535e271ebd45c11eb072efde4cc9885b25ba5c39f91d' #rtkiow8x64.sys
+            - '65329dad28e92f4bcc64de15c552b6ef424494028b18875b7dba840053bc0cdd' #segwindrvx64.sys
+            - 'f8430bdc6fd01f42217d66d87a3ef6f66cb2700ebb39c4f25c8b851858cc4b35' #superbmc.sys
+            - '9f1229cd8dd9092c27a01f5d56e3c0d59c2bb9f0139abf042e56f343637fda33' #semav6msr.sys
+            - 'b03f26009de2e8eabfcf6152f49b02a55c5e5d0f73e01d48f5a745f93ce93a29' #piddrv64.sys
+    driver_img:
+        ImageLoaded|endswith:
```

Comment:
I wouldn't include the names, since they could use the same name for a non-vulnerable new version of that driver

Reply:
Done. I removed the commented names

Reply:
Sorry, I meant all names that do not include a version number in their name, which is most of them. 
Is the reason for this understandable? 
I think I'm not wrong assuming that fixed versions may have been published later under the same name of the vulnerable driver. Or am I too anxious?

Reply:
Yes, later versions have been updated and still have the same names, as I understand.

My idea was to put a comment in front of the hash that way the reader can know which hash is related to which driver. Perhaps he wants to remove a specific one because he uses it, that way It would be easier to do.

Reply:
Ah, yes. I see.

I'd say no one wants to use a vulnerable driver. The hashes are great, because they address that very exact version of a driver. Whenever a vulnerable driver gets loaded, an analyst should check the hash (and the name in the event) and see if it accepted (unlikely). 
I don't think that the names are names are required in the rule. They should appear in the log line on which the Sigma rule has matched.

Reply:
Great that means, I don't have to go and copy 300 driver names 😅 GG